cli/server.ts

```diff
@@ -1102,26 +1111,33 @@ export function serveAsync(options: ServeOptions) {
 
         if (webFile) {
             if (/\.html$/.test(webFile)) {
-                let html = expandHtml(fs.readFileSync(webFile, "utf8"))
+                let html = expandHtml(fs.readFileSync(webFile, "utf8"), htmlParams)
                 sendHtml(html)
             } else {
                 sendFile(webFile)
             }
         } else {
             const m = /^\/(v\d+)(.*)/.exec(pathname);
             if (m) pathname = m[2];
-            const md = readMd(pathname);
-            const mdopts = <pxt.docs.RenderOptions>{
-                template: expandDocFileTemplate("docs.html"),
-                markdown: md,
-                theme: pxt.appTarget.appTheme,
-                filepath: pathname,
-                TOC: resolveTOC(pathname)
-            };
-            let html = pxt.docs.renderMarkdown(mdopts)
-            sendHtml(html, U.startsWith(md, "# Not found") ? 404 : 200)
+            const lang = opts["translate"] ? "pxt" : opts["lang"] as string;
```

Comment:
@darzu backend support needed for this route

Reply:
So if I'm understanding this right, one of the backend changes needed is simply:
If request comes in with translate=1 q.s., then set "incontexttranslations" in pubinfo

It looks like the backend uses renderMarkdown from backend utils which has logic to turn on/off sections via @ifdef

Reply:
Yes correct. The backends need to set incontexttranslations is translate == 1 and use the "pxt" locale